dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.0.json

```diff
@@ -1511,19 +1556,12 @@
        "type": "html",
        "name": "addresse_dont_match_text",
        "visible": false,
-       "visibleIf": "{gift_card_email} <> {gift_card_email_verify}",
+       "visibleIf": "{gift_card_email} <> {gift_card_email_verify} and {gift_card_email_verify} notempty",
        "html": "<p><span style='color: #800000;'>&nbsp;<strong>WARNING</strong>: The two addresses don't match!</span></p>"
-      },
-      {
-       "type": "html",
-       "name": "Warning",
-       "visible": false,
-       "visibleIf": "{gift_card_email} = {gift_card_email_verify}",
-       "html": "<p><span style='color: #339966;'>&nbsp;Addresses match!</span></p>"
       }
      ],
      "visibleIf": "{gift_card_yes_no} = 'yes'",
-     "title": "Gift Card Email Panel"
+     "title": "Gift Card Email Verification"
```

Comment:
We inconsistently refer to both "gift card" and "gift code" across the survey.  Should we settle on one term?

Reply:
changed to card